### PR TITLE
Rename to --media-range-track-background-color

### PR DESCRIPTION
**Pull request by luwes**
Related to #141
Rename `--media-range-track-background` to `--media-range-track-background-color`

### CODE REVIEW

File: docs/styling.md [hunk 1]
```diff
@@ -36,26 +36,26 @@ Our current styling architecture is still quite nascent and is very likely to un
...
-| `--media-range-track-box-shadow`    | `box-shadow`       | `none`                    | box shadow of the underlying slider's track display                                                                        | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
-| `--media-range-track-transition`    | `transition`       | `none`                    | apply a [transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition) to the underlying slider's track display | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
+| Name                                   | CSS Property       | Default Value             | Description                                                                                                                | Notes                                                                                                                                                                  |
```
Comment (heff) [inline docs/styling.md:39]: Oh man, this diff is not helpful. 😆 I'm assuming there's nothing critical happening here though.

**Review by heff — APPROVED** [review]
It's worth checking if we're using this in mux-player anywhere.

**Comment by heff** [comment]
Can you check the merge conflict? Is it that another var was changed in the table?

**Comment by luwes** [comment]
good call, we're using this in the mux-player's! heads up @cjpillsbury if this gets merged

**Review by heff — APPROVED** [review]

**Pull request merged**